src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressionManagerInitInfo.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Acuminator.Utilities.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Acuminator.Utilities.DiagnosticSuppression
+{
+	public readonly struct SuppressionManagerInitInfo : IEquatable<SuppressionManagerInitInfo>
+	{
+		public string Path { get; }
+
+		public bool GenerateSuppressionBase { get; }
+
+		public SuppressionManagerInitInfo(string path, bool generateSuppressionBase)
+		{
+			path.ThrowOnNullOrWhiteSpace(nameof(path));
+
+			Path = path;
+			GenerateSuppressionBase = generateSuppressionBase;
+		}
+
+		public override bool Equals(object obj) => obj is SuppressionManagerInitInfo initInfo
+			? Equals(initInfo)
+			: false;
+
+		public bool Equals(SuppressionManagerInitInfo other) => 
+			GenerateSuppressionBase == other.GenerateSuppressionBase && Path == other.Path;
```

Comment:
Why? The `Path` is a path to the file which is case-sensitive and I don't see any code which is case-insensitive

Reply:
Indeed it case-insensitive( Ok, I'll fix it

Reply:
done. Also this means that current implementation is more "correct" than previous one via value tuple because value tuple will use case sensitive string comparison